jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannel.java

```diff
@@ -82,9 +84,11 @@
     private final HttpChannelState _state;
     private final Request _request;
     private final Response _response;
+    private final Collection<Listener> _durableListeners;
+    @Deprecated
+    private final List<Listener> _cyclicListeners = new ArrayList<>();
```

Comment:
Renamed `cyclicListeners` to `transientListeners` - seems to more appropriately describe their contrast with `durableListeners`.